### PR TITLE
Replace media paragraph

### PR DESCRIPTION
**Pull request by chrfritsch**
Make sure these boxes are checked before submitting your pull request - thank you!

- [x] All tests are running locally. ([How to run the test?](https://github.com/BurdaMagazinOrg/thunder-distribution/blob/8.x-1.x/docs/development.md#how-to-run-the-tests))
- [x] Necessary update hooks are provided.
- [x] User roles have correct access for new introduced permission.
- [x] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [x] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [x] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

File: modules/thunder_article/config/install/field.field.node.article.field_paragraphs.yml [hunk 1]
```diff
@@ -42,8 +44,8 @@ settings:
         enabled: true
         weight: 3
       media:
-        enabled: true
         weight: 4
+        enabled: false
```
Comment (dbosen) [inline modules/thunder_article/config/install/field.field.node.article.field_paragraphs.yml]: Could be deleted

File: modules/thunder_media/config/install/views.view.video_browser.yml [hunk 2]
```diff
@@ -387,42 +386,35 @@ display:
...
             remember: false
             multiple: false
             remember_roles:
               authenticated: authenticated
               anonymous: '0'
-              editor: '0'
-              seo: '0'
-              administrator: '0'
-            reduce: true
+            reduce: false
```
Comment (dbosen) [inline modules/thunder_media/config/install/views.view.video_browser.yml]: Why did that change?

File: tests/src/FunctionalJavascript/ArticleCreationTest.php [hunk 3]
```diff
@@ -34,8 +34,8 @@ public function testCreateArticle() {
 
     $this->selectMedia('field_teaser_media', 'image_browser', ['media:1']);
 
-    // Add Media Image Paragraph.
-    $this->addMediaParagraph(static::$paragraphsField, ['media:5']);
+    // Add Image Paragraph.
+    $this->addImageParagraph(static::$paragraphsField, ['media:5']);
```
Comment (dbosen) [inline tests/src/FunctionalJavascript/ArticleCreationTest.php]: addVideoParagraph test is missing

File: modules/thunder_updater/updates.yml [hunk 4]
```diff
@@ -20,3 +20,7 @@ v1.1:
...
         '#description': '<p>Language filtering should be removed in views used by media browsers. Media entities don''t have relation to language and filter doesn''t have effect.</p><p>In order to do that, you should make following steps:<ol><li>Go on views administration page.</li><li>Search for view that you want to edit (fe. "Image Browser", "Media Browser").</li><li>Click <b>Edit</b> to modify view.</li><li>On view editing page under <b>FILTER CRITERIA</b> find link <b>Media: Translation language (exposed)</b> and click it.</li><li>In pop-up dialog click <b>Remove</b> in bottom action bar.</li><li>Save view changes by clicking <b>Save</b> button.</li></ol> </p><p>Repeat described process for all views you need.</p>'
         views_page: { '#text': 'Views administration page', '#url': '/admin/structure/views' }
+    v1_1__image_video_paragraph:
+        '#title': 'Splitting media paragraph into video and image paragraph'
+        '#description': '<p>We think for editors it is more clearer to have a seperate video and image paragraph. So we installed that in an update hook for you. You can now decide by yourselve, if you want to use them.</p>'
```
Comment (dbosen) [inline modules/thunder_updater/updates.yml]: Checkbox is missing for the actual import of the paragraphs.

Comment (dbosen) [inline modules/thunder_updater/updates.yml]: It should be made clear, that the  video and image paragraphs are meant as an alternative to the previously existing media paragraph.

File: modules/thunder_media/thunder_media.install [hunk 5]
```diff
@@ -531,3 +531,68 @@ function thunder_media_update_8101() {
...
+
+  // Output log result depending on channel used for update execution.
+  if (function_exists('drush_log') && PHP_SAPI === 'cli') {
+    $updateLogger->outputDrush();
+
+    return '';
+  }
+
+  return $updateLogger->outputHtml();
+}
```
Comment (dbosen) [inline modules/thunder_media/thunder_media.install]: Add 'access video_browser entity browser pages' to editor and seo role in the update hook.

**Comment by chrfritsch** [comment]
Update tests are failing now

**Pull request merged**